homeassistant/components/mazda/device_tracker.py

```diff
@@ -22,32 +22,20 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 class MazdaDeviceTracker(MazdaEntity, TrackerEntity):
     """Class for the device tracker."""
 
-    @property
-    def name(self):
-        """Return the name of the entity."""
-        vehicle_name = self.get_vehicle_name()
-        return f"{vehicle_name} Device Tracker"
-
-    @property
-    def unique_id(self):
-        """Return a unique identifier for this entity."""
-        return self.vin
+    def __init__(self, client, coordinator, index) -> None:
+        """Initialize Mazda device tracker."""
+        super().__init__(client, coordinator, index)
 
-    @property
-    def icon(self):
-        """Return the icon to use in the frontend."""
-        return "mdi:car"
+        self._attr_name = f"{self.vehicle_name} Device Tracker"
+        self._attr_unique_id = self.vin
+        self._attr_icon = "mdi:car"
+        self._attr_force_update = False
```

Comment:
These last two you could set at the class level, above the `__init__` method, since they're always the same.

Reply:
Thanks, fixed.